pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2430,10 +2430,6 @@ public CompletableFuture<Void> onPoliciesUpdate(Policies data) {
                 CompletableFuture<Void> persistentPoliciesFuture = checkPersistencePolicies();
                 // update rate-limiter if policies updated
                 dispatchRateLimiter.ifPresent(DispatchRateLimiter::updateDispatchRate);
-                if (this.subscribeRateLimiter.isPresent()) {
```

Comment:
Can you point out where is this repeated with?

Reply:
> Can you point out where is this repeated with?

Look good